src/ext/java/org/opentripplanner/ext/vectortiles/layers/areastops/AreaStopPropertyMapper.java

```diff
@@ -0,0 +1,52 @@
+package org.opentripplanner.ext.vectortiles.layers.areastops;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import org.opentripplanner.apis.support.mapping.PropertyMapper;
+import org.opentripplanner.framework.i18n.I18NStringMapper;
+import org.opentripplanner.inspector.vector.KeyValue;
+import org.opentripplanner.transit.model.network.Route;
+import org.opentripplanner.transit.model.site.AreaStop;
+import org.opentripplanner.transit.model.site.StopLocation;
+import org.opentripplanner.transit.service.TransitService;
+
+public class AreaStopPropertyMapper extends PropertyMapper<AreaStop> {
+
+  private final Function<StopLocation, Collection<Route>> getRoutesForStop;
+  private final I18NStringMapper i18NStringMapper;
+
+  protected AreaStopPropertyMapper(
+    Function<StopLocation, Collection<Route>> getRoutesForStop,
+    Locale locale
+  ) {
+    this.getRoutesForStop = getRoutesForStop;
+    this.i18NStringMapper = new I18NStringMapper(locale);
+  }
+
+  protected static AreaStopPropertyMapper create(TransitService transitService, Locale locale) {
+    return new AreaStopPropertyMapper(transitService::getRoutesForStop, locale);
+  }
+
+  @Override
+  protected Collection<KeyValue> map(AreaStop stop) {
+    var routeColors = getRoutesForStop
+      .apply(stop)
+      .stream()
+      .map(Route::getColor)
+      .filter(Objects::nonNull)
+      .distinct()
+      // the MVT spec explicitly doesn't cover how to encode arrays
+      // https://docs.mapbox.com/data/tilesets/guides/vector-tiles-standards/
+      .collect(Collectors.joining(","));
```

Comment:
This part always confuses me and I forget how it should be. Seems like we have previously used `JSONArray.toJSONString` to encode lists.

Reply:
It's a mess. We also have https://github.com/opentripplanner/OpenTripPlanner/blob/63327b72b243052806602a5fbd8d3b7c9ec4b70e/src/ext/java/org/opentripplanner/ext/vectortiles/layers/vehiclerental/mapper/DigitransitVehicleRentalStationPropertyMapper.java#L45-L53

It seems on all the layers that I wrote I've just used comma-separated lists. :facepalm: 

Not sure what is appropriate, since MVT is explicitly silent: https://docs.mapbox.com/data/tilesets/guides/vector-tiles-standards/#how-to-encode-attributes-that-arent-strings-or-numbers